integration_tests/support/sharedTests.ts

```diff
@@ -78,6 +82,23 @@ const courseParticipationPresenter2 = {
   }),
   addedByDisplayName: StringUtils.convertToTitleCase(user.name),
 }
+const offenceCodeOne = offenceDtoFactory.build({ code: 'O1', description: 'Offence 1' })
+const offenceCodeTwo = offenceDtoFactory.build({ code: 'O2', description: 'Offence 2' })
+const offenderBooking = inmateDetailFactory.build({
```

Comment:
I think this is just a Prison API thing, but I'm getting confused (again?) about the difference between and offender booking and an inmate detail 🤔

Reply:
I've also experienced the confusion 😀 .  The Prison API offender booking endpoint that we use returns a type of `InmateDetail`.   https://prison-api-dev.prison.service.justice.gov.uk/swagger-ui/index.html#/bookings/getOffenderBookingByOffenderNo. 

At the time I thought it was best to match the name of the factory to the name of the return type, but could be open to renaming it to `offenderBookingFactory`